test/system/broadcasts_test.rb

```diff
@@ -23,6 +23,16 @@ class BroadcastsTest < ApplicationSystemTestCase
     assert_selector("#messages-count", text: Message.count, wait: 10)
   end
 
+  test "New messages update the message count with inline: content" do
+    visit messages_path
+    
+    message = Message.new(record_id: 1, content: "Message with inline")
+    message.broadcast_update_to :messages, target: "messages-count", 
+      inline: ERB.new("<%= Message.count %> messages sent").result
```

Comment:
The `inline:` option accepts the ERB _template string_ itself. What this test is covering is essentially the same as the `html:` option.

You can pass the ERB template directly, without the wrapping `ERB.new` and `.result` call. I've edited [my suggestion](https://github.com/hotwired/turbo-rails/pull/245/files/e9c53b80a73f01d004bf216709ad6aafdf5620dc..850c17641f57d01fcc53303c07d6b8056d4f47f2#r714795322) slightly since the first time I posted it, if you want a reference.

Reply:
Okay I understand now thanks

Reply:
Actually Im thinking maybe the test is just flaky because were testing the same element and using broadcasts to update Ill change the erb test to update a separate element

Reply:
The system tests are flaky, I agree. I've opened https://github.com/hotwired/turbo-rails/pull/248 to try and help with that.

On top of that, your `broadcast_update_to` calls are not wrapped in `assert_appends_text` blocks (which is renamed to `assert_broadcasts_text` in #248), so the underlying broadcast jobs won't execute, so the `<turbo-stream>` elements won't be pushed over Action Cable.

Reply:
@yunggindigo if you rebase off `main` and throw away your branch's `Message` changes, these tests should be more predictable. #248 has merged, and `Message` is a full Active Record model.